stable/hadoop/templates/_helpers.yaml

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "name" -}}
```

Comment:
We've just started namespacing templates. Please change them like so and apply throughout the chart:

`{{- define "hadoop.name" -}}`
`{{- define "hadoop.fullname" -}}`

Reply:
The output of `helm create` with v2.6.1 still generates helpers without the namespace.